### PR TITLE
Code readability

### PR DESCRIPTION
**Pull request by madhavshah49**
### Description

This pull request is to address ...

Fixes #<Issue Number>

### CODE REVIEW

File: docs/spectrum/spectrum.rst [hunk 1]
```diff
@@ -330,7 +329,7 @@ Use :py:meth:`~radis.spectrum.spectrum.Spectrum.plot_populations`::
 
 .. _label_spectrum_linesurvey:
 
-Plot line survey
+Plotline survey
```
Comment (erwanp) [inline docs/spectrum/spectrum.rst:332]:
```suggestion
Plotline survey
```

not sure of this one !

File: docs/spectrum/spectrum.rst [hunk 2]
```diff
@@ -247,15 +246,15 @@ can be retrieved in arbitrary units with the :meth:`~radis.spectrum.spectrum.Spe
 method.
 
 When a spectral unit is convolved with :meth:`~radis.spectrum.spectrum.Spectrum.apply_slit`,
-a new convolved spectral quantities is created. The unit of the convolved quantity may be different,
-depending on how the slit function was normalised. Several options are available in RADIS.
+a new convolved spectral quantity is created. The unit of the convolved quantity may be different,
```
Comment (erwanp) [inline docs/spectrum/spectrum.rst:249]:
```suggestion
a new convolved spectral array is created. The unit of the convolved spectral array may be different,
```

the name is being changed in #236

File: docs/spectrum/spectrum.rst [hunk 3]
```diff
@@ -524,7 +523,7 @@ change the unit::
 
 .. _label_spectrum_algebra:
 
-Multiply, substract
+Multiply, subtract
 -------------------
```
Comment (erwanp) [inline docs/spectrum/spectrum.rst:527]:
```suggestion
------------------
```

alignment

**Review by erwanp — CHANGES_REQUESTED** [review]

**Comment by erwanp** [comment]
@madhavshah49 have a look, couple minor change requests + one conflict

**Comment by madhavshah49** [comment]
@erwanp hey I've done the minor changes sorry did not create the pull request yet and the conflict you want me to add something to the file or just redraft it as it was earlier

**Comment by madhavshah49** [comment]
@erwanp hey I've done the minor changes sorry it took a while and did not create the pull request yet, also in the conflict do you want me to add something to the file or just redraft it as it was earlier

**Comment by madhavshah49** [comment]
@erwanp hey I've done the minor changes sorry it took a while, also in the conflict do you want me to add something additional to the file or just redraft it as it was earlier

**Comment by erwanp** [comment]
No worries. You can fix the conflict by picking the option that uses "array"  not "quantity"

**Comment by madhavshah49** [comment]
@erwanp Ready to merge?

**Comment by erwanp** [comment]
Can you fix the conflict?

File: docs/spectrum/spectrum.rst [hunk 4]
```diff
@@ -247,15 +246,15 @@ can be retrieved in arbitrary units with the :meth:`~radis.spectrum.spectrum.Spe
...
+depending on how the slit function was normalized. Several options are available in RADIS.
 Please refer to the documentation of the :meth:`~radis.spectrum.spectrum.Spectrum.apply_slit` method.
 
 
 
 
 
-How to access a Spectrum properties?
+How to access Spectrum properties?
 ====================================
```
Comment (erwanp) [inline docs/spectrum/spectrum.rst:258]:
```suggestion
How to access Spectrum properties?
==================================
```

File: docs/spectrum/spectrum.rst [hunk 5]
```diff
@@ -275,7 +274,7 @@ method to retrieve the quantity un the units you want::
 See :ref:`spectral arrays <label_spectral _arrays>` for the list
 of spectral arrays.
 
-Get wavelength / wavenumber
+Get wavelength/wavenumber
 ---------------------------
```
Comment (erwanp) [inline docs/spectrum/spectrum.rst:278]:
```suggestion
Get wavelength/wavenumber
-------------------------
```

File: docs/spectrum/spectrum.rst (hunk 1 above)
Comment (erwanp) [inline docs/spectrum/spectrum.rst:332]:
```suggestion
Plot Line survey
```

**Review by erwanp — CHANGES_REQUESTED** [review]
Fix format of titles, and make sure we use "line survey" and not "plotline survey" (@madhavshah49  you can accept everything from the Web interface with Batch commits !)

**Review by erwanp — APPROVED** [review]

**Pull request merged**